cmd/lastZWriter/lastZWriter.go

```diff
@@ -30,28 +30,54 @@ func MakeArray(lastZ string, pairwise string, speciesListFile string, refListFil
 				if parameters == nil || matrix == "" {
 					log.Fatalf("Reference %s and species %s returned no parameters or matrix.", refList[ref], speciesList[spec])
 				}
-				allLines = writeFile(lastZ, pairwise, refList[ref], speciesList[spec], parameters, matrix, allLines)
+				allLines = writeFile(lastZ, pairwise, refList[ref], speciesList[spec], parameters, matrix, targetModifier, allLines)
 			}
 		}
 	}
 	fileio.Write(outText, allLines)
 }
 
-func writeFile(lastZ string, pairwise string, reference string, species string, parameters []string, matrix string, allLines []string) (lines []string) {
+// MakeArraySimple has no allDists, m, or mPath (will not generate matrix), and parameters is directly input as 1 string not generated as []string
+func MakeArraySimple(lastZ string, pairwise string, speciesListFile string, refListFile string, parameters string, outText string, targetModifier string) {
+	speciesList := fileio.Read(speciesListFile)
+	refList := fileio.Read(refListFile)
+	fileio.EasyCreate(outText)
+	var allLines []string
+	for ref := range refList {
+		for spec := range speciesList {
+			match := strings.Compare(speciesList[spec], refList[ref])
+			if match != 0 {
```

Comment:
I think we could get rid of the match variable and just do "if speciesList[spec] != refList[ref] {"